src/interpreter/runner.ts

```diff
@@ -261,26 +261,26 @@ async function runMemberCall(memberCall: MemberCall, context: RunnerContext): Pr
     }
 
     if (memberCall.explicitOperationCall) {
-        if (isFunctionDeclaration(ref)) {
+        const targetFunction = isFunctionDeclaration(ref) ? ref : (isFunctionDeclaration(value) ? value : undefined);
+        if (targetFunction) {
```

Comment:
Suggestion: Given that we assign `value = ref` in case `ref` is a function declaration, we can use `isFunctionDeclaration(value)` directly.
```suggestion
        if (isFunctionDeclaration(value)) {
```

Reply:
You are so right, many thanks for the hint, much simpler!
Will commit the change.